### PR TITLE
Override default settings instead of final settings

### PR DESCRIPTION
**Pull request by leplatrem**
My proposition instead of #342

### CODE REVIEW

**Comment by Natim** [comment]
Cool :+1:

**Comment by leplatrem** [comment]
As usual, I was not able to think properly until I put a hand on some code :((

@Natim I haven't tested with Kinto though, but it looks consistent

**Comment by Natim** [comment]
I don't know if we can add a test to make sure default_settings (both from cliquet and the app) are overridden by the config.ini file.

**Comment by leplatrem** [comment]
That's what I was telling you here: https://github.com/mozilla-services/kinto/pull/90#issuecomment-117616892

Settings of ``[app:main]`` section of ``.ini`` arrive in ``**settings`` arg of ``main()``. https://github.com/mozilla-services/kinto/blob/master/kinto/__init__.py#L28-L29

**Comment by Natim** [comment]
Ok perfect :)

File: cliquet/initialization.py [hunk 1]
```diff
@@ -340,21 +341,25 @@ def initialize(config, version=None, project_name=None, default_settings=None):
         in application settings.
     :param dict default_settings: Override cliquet default settings values.
     """
+    cliquet_defaults = cliquet.DEFAULT_SETTINGS.copy()
+
     if default_settings:
-        config.add_settings(default_settings)
+        cliquet_defaults.update(default_settings)
```
Comment (ametaireau) [inline cliquet/initialization.py]: Thanks :)

**Pull request merged**